lib/subscribers/mysql/helper.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const dbutils = require('../../db/utils')
+const properties = require('../../util/properties')
+const symbols = require('../../symbols')
+const DatastoreParameters = require('../../shim/specs/params/datastore')
+
+/**
+ * Extracts the query string and callback position
+ * from the function arguments.
+ * @param {object[]} args the original query arguments
+ * @returns {object} { query, callback }
+ */
+function extractQueryArgs(args) {
```

Comment:
instead of helpers what about having some mysql-connection class that has these methods?